sandbox/101functions/basics/host.md

```diff
@@ -0,0 +1,37 @@
+## Host.json basics
+
+At the root of an application is a metadata file that describes the global configuration of the host serving the different functions.
```

Comment:
> metadata file that describes the global configuration of the host

From this, I can infer you're talking about `host.json`, but it might not be obvious to others. Maybe something like: " ...application is a metadata file called host.josn" or "Present at the root of the application, host.json is a metadata file..."